src/config/consts.js

```diff
@@ -41,6 +41,7 @@ export const consts = {
     mail: '(E-Mail-Adresse)',
     mailHint: '(Wechselt zur E-Mail-App)',
     mapHint: '(Wechselt zur Karten-App)',
+    menuItem: 'Menüelement (Taste)',
```

Comment:
i would not create a new "context" here with `Menü`. i relates to the dropdown, right? so what about `dropDownMenuItem: Dropdown-Menü-Eintrag (Taste)`?

Reply:
yes you are right. It's better now.
https://github.com/smart-village-solutions/smart-village-app-app/commit/0a6c8ce7fa5ed299446fb3ab7a910a08b673a271